doctorapp/src/components/home/Appointments.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+    <div class="appointments">
+      <h1>Appointments</h1>
+      <Appointment v-for="appointment in appointments" :key="appointment.time" :time="appointment.time" :date="appointment.date" :type="appointment.type" :location="appointment.location" :provider="appointment.provider" >
```

Comment:
that's one hell of a long line, do your lint settings not get mad?
```suggestion
      <Appointment 
            v-for="appointment in appointments"
            :key="appointment.time"
            :time="appointment.time"
            :date="appointment.date"
            :type="appointment.type"
            :location="appointment.location"
            :provider="appointment.provider"
      >
```